pages/common/kubectl.md

```diff
@@ -34,3 +26,11 @@
 - Display an explanation of the specific field:
 
 `kubectl explain pods.spec.containers`
+
+- Print the logs for a container in a pod or specified resource:
+
+`kubectl logs my-pod`
+
+- Run command in an existing pod:
+
+`kubectl exec my-pod -- ls /`
```

Comment:
Same here ^^

Reply:
Same here for wrapping (lol)

Reply:
Just to be clear, please wrap both {{my_pod}} and {{ls /}}